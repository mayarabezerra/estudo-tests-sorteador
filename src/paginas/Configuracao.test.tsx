import { render } from '@testing-library/react'
import React from 'react'
import { RecoilRoot } from 'recoil'
import Configuracao from './Configuracao'


const mockNavegacao = jest.fn()

jest.mock('react-router-dom', () => {
    return {
        useNavigate: () => mockNavegacao
    }
})

describe('quando não existem participantes suficientes', () => {
    test('a brincadeira não pode ser iniciada', () => {
        const {container} = render (
            <RecoilRoot>
                <Configuracao />
            </RecoilRoot>)

            expect(container).toMatchSnapshot()

    })
})