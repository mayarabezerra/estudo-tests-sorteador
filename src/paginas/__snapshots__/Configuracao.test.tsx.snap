// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`quando não existem participantes suficientes a brincadeira não pode ser iniciada 1`] = `
<div>
  <form>
    <input
      placeholder="Insira os nomes dos participantes"
      type="text"
      value=""
    />
    <button
      disabled=""
    >
      Adicionar
    </button>
    
  </form>
  <ul />
  <footer>
    <button
      disabled=""
    >
      Iniciar brincadeira
    </button>
  </footer>
</div>
`;
